confignode/src/main/java/org/apache/iotdb/confignode/service/thrift/server/ConfigNodeRPCServerProcessor.java

```diff
@@ -152,6 +156,25 @@ public DeviceGroupHashInfo getDeviceGroupHashInfo() throws TException {
     return null;
   }
 
+  @Override
+  public TSStatus createUser(CreateUserReq req) throws TException {
```

Comment:
maybe only two rpc methods  for authorization is ok?  one for create user/role, one for query user/role/priviliege?